common/config/config.go

```diff
@@ -577,6 +577,7 @@ type (
 		// Signing key provider for validating JWT tokens
 		JWTKeyProvider       JWTKeyProvider `yaml:"jwtKeyProvider"`
 		PermissionsClaimName string         `yaml:"permissionsClaimName"`
+		PermissionsPattern   string         `yaml:"permissionsPattern"`
```

Comment:
nit: consider calling this `PermissionsRegex` to clearly indicate what the contents should be and update the naming throughout.